docs/token_authorization_api.md

```diff
@@ -0,0 +1,50 @@
+# TorchServe token authorization API
+
+## Configuration
+1. Enable token authorization by adding the provided plugin at start using the `--plugin-path` command.
```

Comment:
It seems that "-plugin-path" in cmd can work in pytest. If yes, please replace with the correct instruction for prod.